CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## [11.4.0](https://github.com/npm/cli/compare/v11.3.0...v11.4.0) (2025-05-13)
+### Features
+* [`57aa89f`](https://github.com/npm/cli/commit/57aa89ff70e0c6186a43888b944b5799b25c7bc8) [#8265](https://github.com/npm/cli/pull/8265) use run by default and run-script as the alias (#8265) (@owlstronaut)
+* [`0d4c023`](https://github.com/npm/cli/commit/0d4c023914f835927540bd0110c1ca5716b84056) [#8234](https://github.com/npm/cli/pull/8234) install: add package info to json output (#8234) (@wraithgar)
+### Bug Fixes
+* [`2210d7a`](https://github.com/npm/cli/commit/2210d7a670ac3522ceee8856a3399e8f44e77bbe) [#8278](https://github.com/npm/cli/pull/8278) powershell: use Invoke-Expression to pass args (#8278) (@alexsch01)
```

Comment:
@wraithgar, please add @mbtools to 8278. Thx 🙏

Reply:
That will have to be a follow up commit post-release.  The changelog is auto generated for each release.  We can post-edit things but can't live-edit if that makes sense.

Adding a `co-authored-by` footer also works ([ref](https://github.com/npm/cli/commit/8669d0931abd0ae4b655cf9e5a024065054a8bb4), but that wont' help here because the commit already landed. Sorry about that.

Reply:
I can add you to the release notes for the [release](https://github.com/npm/cli/releases/tag/v11.3.0) which will at least make it right there, and I believe it's what powers the PR into the nodejs repo.

Reply:
Thanks